src/bin/pjv.ts

```diff
@@ -1,54 +1,53 @@
-#!/usr/bin/env node
```

Comment:
🐛 This shouldn't be removed; the file gets output to `lib/bin/pjv.mjs` which is a `bin` entry in `package.json`... 🤔